.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishMergedExtractorConfiguration.cs

```diff
@@ -64,6 +64,38 @@ public class EnglishMergedExtractorConfiguration : BaseDateTimeOptionsConfigurat
 
             // (the)? (day|week|month|year)
             new Regex(DateTimeDefinitions.SingleAmbiguousTermsRegex, RegexFlags),
+
+        };
+
+        // Skip Regexes under TasksMode
+        public static readonly Regex[] TasksModeMentionFilters =
```

Comment:
As mentioned, I'd prefer is this is not automatically created always. Only if the mode is enabled.
Perhaps a flag check in the constructor and then call a build method?

Reply:
I have made changes, now when tasks Mode sets to true, then only TasksModeMentionFilters will get created.

Reply:
Can you delete this commented our static block?

Reply:
deleted comment block.